src/css/custom.css

```diff
@@ -95,8 +95,6 @@ html {
   .navbar .navbar-sidebar__items {
     flex: 1 1 auto;
     display: flex;
-    align-items: center;
```

Comment:
This is the only actual change. Everything else was an autoformat that I didn't catch.